java-client/session-dagger/src/test/java/io/deephaven/client/impl/TableServiceAsyncTest.java

```diff
@@ -93,8 +93,11 @@ private static List<TableSpec> createLongChain(int numColumns, int numRows) {
             if (i == 0) {
                 longChain.add(TableSpec.empty(numRows).view("I_0=ii"));
             } else {
+                // create a chain using the previous table even though we don't need the data
                 final TableSpec prev = longChain.get(i - 1);
-                longChain.add(prev.updateView("I_" + i + " = 1 + I_" + (i - 1)));
+                // to avoid timeouts due to compilation variance reuse the same expression, including the destination
+                // column which shows up in the generated formula's FormulaEvaluationException
+                longChain.add(prev.updateView("I_1 = 1 + I_0"));
```

Comment:
I think reworking as `prev.updateView("Next = Current + 1", "Current = Next")` better matches the original intentions of the query.

Reply:
The test currently does not validate the data. As far as I can tell, the purpose is to ensure that some table returns at the end of the chain.

I like the proposed change as it eliminates the need to recompile the formulas repeatedly while preserving the original implementation. Though, perhaps we should be adding an assert table equals to validate the expected data is returned.

Reply:
Shouldn't my proposal not need repeated compilation? The strings are fixed.

Reply:
> I like the proposed change [...]

I meant to reference your suggestion which is a nice improvement.